src/index.ts

```diff
@@ -279,6 +280,6 @@ class HuddlySdk extends EventEmitter {
     await this.deviceDiscoveryApi.initialize();
   }
 }
-export { CameraEvents };
+export { CameraEvents, Api };
```

Comment:
Why do you need to export this?

Reply:
So I can gain access to the static functions encode/decode from the cli. I'm putting functionality in the cli that should not be accessible to the public in th sdk, e.g. audio readout through hlink.